content/blog/entries/cc-vocabulary-docs-updates-3/contents.lr

```diff
@@ -0,0 +1,64 @@
+
+title: Vocabulary Site Updates (Part 3/n)
+---
+categories:
+cc-vocabulary
+gsod
+gsod-2020
+---
+author: nimishbongale
+---
+series: gsod-2020-vocabulary-usage-guide
+---
+pub_date: 2020-11-25
+---
+body:
+Excited to know more about this week's vocabulary site updates? Read on to find out!
+
+## Vocabulary Site Updates (Edition 3/many more to come)
+
+### What I've been up to
+
+<center>
+<img alt"Halfway There" src="merged.png"/><br>
+<small class="muted">The surreal feeling...</small>
+</center>
+
+Merged? Yes. **Merged**. Here's my story!
+
+- After getting a thumbs up from the UX Designer, I put up my [GSoD Website PR](https://github.com/creativecommons/vocabulary/pull/747) for review.   
+- I was confident there would be changes, and I let them roll in. It's important to note here that what seems perfect to you may not be so to others, and only experience teaches you the right from the wrong.
+- There were a few of them, mainly dealing with spacing, textual content and colors. I resolved them as soon as I could.
+-  [zackkrida](https://opensource.creativecommons.org/blog/authors/zackkrida/) has been kind enough to point and enumerate all of them for me!
+- After receiving a final approval from the engineering team, my PR was finally merged!
+- The final draft of the vocabulary site is live! It will soon be deployed (on [netlify](https://netlify,com)) and be made available for public viewing.
```

Comment:
```suggestion
- The final draft of the vocabulary site is live! It will soon be deployed (on [Netlify](https://netlify,com)) and be made available for public viewing.
```